apps/email/js/cards/base_card.js

```diff
@@ -1,20 +1,24 @@
 'use strict';
+/* global mozIntl */
 define(function(require) {
   var mozL10n = require('l10n!'),
       base = require('./base');
 
   // Set up the global time updates for all nodes.
   (function() {
-    var formatter = new mozL10n.DateTimeFormat();
     var updatePrettyDate = function updatePrettyDate() {
-      var labels = document.querySelectorAll('[data-time]');
-      var i = labels.length;
-      while (i--) {
-        labels[i].textContent = formatter.fromNow(
-          labels[i].dataset.time,
+      var labels = [...document.querySelectorAll('[data-time]')];
+
+      labels.forEach(label => {
+        mozIntl._gaia.relativeDate(
```

Comment:
Shoo! I didn't ask for a review yet for a reason! :)

No, but seriously, yeah, I will definitely optimize it for batch operations. :)

I need to land a couple patches before I can submit this (mozIntl extension is a separate patch and has its own bug). I expect to submit it for review around Wed.

Thanks for the suggestion!